crates/ef-testing/src/models/case.rs

```diff
@@ -216,6 +216,9 @@ impl Case for BlockchainTestCase {
         let specific_tests_to_run: Option<HashSet<String>> = std::env::var("RUN_SPECIFIC_TESTS")
             .ok()
             .map(|test_names| test_names.split(',').map(String::from).collect());
+        let specific_file_to_run: Option<HashSet<String>> = std::env::var("RUN_SPECIFIC_FILES")
```

Comment:
an idea of how we can achieve the same thing with a single environment variable, i.e. having the ability to do

```
RUN_SPECIFIC_TESTS=jumpi_d14g0v0_Shanghai cargo test -p ef-testing --features ef-tests -- --nocapture
```

or 

```
RUN_SPECIFIC_TESTS=jumpi* cargo test -p ef-testing --features ef-tests -- --nocapture

```

- instead of parsing one or many strings, we treat input as regex
- instead of checking if a string is in a set, we check if the test_name matches the regex

wdyt?

Reply:
Let me try this!

Reply:
and maybe then only `TEST=jumpi ...` or `TARGET=jumpi ...` instead of the long `RUN_SPECIFIC_TEST`